test/unit/adb-specs.ts

```diff
@@ -30,5 +30,110 @@ describe('ADB', function () {
       expect(clone.remoteAdbHost).to.equal('example.com');
       expect(clone.adbHost).to.not.equal(original.adbHost);
     });
+
+    describe('-a option', function() {
+      it('should not have -a option', function () {
```

Comment:
I think AI is usually too talkative regarding the amount of tests. Can we only keep ones that are actually significant?